src/components/atoms/Textarea/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { TextareaHTMLAttributes } from "react";
+
+export interface TextareaProps
+  extends TextareaHTMLAttributes<HTMLTextAreaElement> {
+    field?: any
+  }
+
+// eslint-disable-next-line react/display-name
```

Comment:
Remove this and add before `export default Textarea;` -> Textarea.DisplayName = 'Textarea'